Mailnag/configuration/accountdialog.py

```diff
@@ -68,12 +71,18 @@ def __init__(self, parent, acc):
 		self._cmb_account_type = builder.get_object("cmb_account_type")
 		self._label_account_name = builder.get_object("label_account_name")
 		self._entry_account_name = builder.get_object("entry_account_name")
+		self._label_account_user = builder.get_object("label_account_user")
 		self._entry_account_user = builder.get_object("entry_account_user")
+		self._label_account_password = builder.get_object("label_account_password")
 		self._entry_account_password = builder.get_object("entry_account_password")
 		self._label_account_server = builder.get_object("label_account_server")
 		self._entry_account_server = builder.get_object("entry_account_server")		
 		self._label_account_port = builder.get_object("label_account_port")
 		self._entry_account_port = builder.get_object("entry_account_port")
+		self._label_account_file_path = builder.get_object("label_account_file_path")
+		self._chooser_account_file_path = builder.get_object("chooser_file_path")
+		self._label_account_directory_path = builder.get_object("label_account_directory_path")
+		self._chooser_account_directory_path = builder.get_object("chooser_directory_path")
```

Comment:
Refactoring idea: All parameter have two widgets, a label and some kind of entry widget (entry, checkbox, chooser), and all these pairs are handled similar ways. The pairs could be wrapped to some kind of parameter objects which all have identical inteface, something like:
```
self._user_entry = EntryParam(builder, "label_account_user", "entry_account_user")
self._file_path_entry = ChooserParam(builder, "label_account_file_path", "chooser_file_path")
...
```
Then all the *Param classes have methods set_value, get_value, set_visible, etc.

I can't explain this better yet, I had to think this a bit more...